core/model/application.go

```diff
@@ -389,7 +389,7 @@ type IdentityProviderSetting struct {
 	Roles  map[string]string `bson:"roles"`  //map[identity_provider_role]app_role_id
 	Groups map[string]string `bson:"groups"` //map[identity_provider_group]app_group_id
 
-	AdminAppAccessRoles []string `bson:"admin_app_access_roles"` //list with the identity provider roles which are approved for admin app account creation
+	AdminAppAccessRoles *[]string `bson:"admin_app_access_roles"` //list with the identity provider roles which are approved for admin app account creation
```

Comment:
Please do not change the model just to be able easier to map what comes from the documentation. As general we do not use pointers when we define arrays in the model.